File: scripts/py-compile-suite/inputs/19_statistics.m
% Descriptive statistics
x = randn(1000, 1);
mu = mean(x);
sigma = std(x);
med = median(x);
mn = min(x);
mx = max(x);
pct = prctile(x, [25, 50, 75]);
v = var(x);

File: scripts/py-compile-suite/outputs/19_statistics.py
import numpy as np

# Descriptive statistics
x = np.random.randn(1000)
mu = np.mean(x)
sigma = np.std(x, ddof=1)
med = np.median(x)
mn = np.min(x)
mx = np.max(x)
pct = np.percentile(x, [25, 50, 75])
v = np.var(x, ddof=1)
